src/ViewModel/Button.php

```diff
@@ -150,6 +151,11 @@ public static function action(
                 ]
             );
             $button->classes .= ' icon icon-' . $variant;
+
+            if ($variant == self::ACTION_VARIANT_COMMENT) {
+                $button->isHypothesisTrigger = true;
+                $button->text = "Comment<span aria-hidden='true'><span data-visible-annotation-count></span> </span><span class='visuallyhidden'>Open annotations (there are currently <span data-hypothesis-annotation-count>0</span> annotations on this page). </span>";
```

Comment:
text can be removed. Journal can supply all of that text.

We need to adjust ButtonTest.php

Reply:
text has been removed and tests have been updated.